<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameColumnsWidget</class>
 <widget class="QWidget" name="RenameColumnsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1503</width>
    <height>884</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: white;
}
QLabel {
    color: #000000;
    font-size: 14px;
    background-color: transparent;
}
QLineEdit {
    padding: 8px 12px;
    border: 1px solid #ced4da;
    border-radius: 4px;
    background-color: #ffffff;
    font-size: 14px;
    min-height: 20px;
    color: #495057;
}
QLineEdit:focus {
    border: 1px solid #80bdff;
    outline: 0;
}
QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}
QPushButton#save_button {
    color: #fff;
    background-color: #007bff;
    border-color: #007bff;
    font-weight: bold;
}
QPushButton#save_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#save_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: bold;
color: #000000;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>컬럼명 변경</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="rename_columns_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: #f8f9fa;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    gridline-color: #e9ecef;
    color: #212529;
}
QTableWidget::item {
    padding: 4px;
    border-bottom: 1px solid #e9ecef;
    color: #212529;
    background-color: white;
}
QTableWidget::item:selected {
    background-color: #e7f5ff;
    color: #212529;
}
QTableWidget::item:focus {
    outline: none;
    border: none;
    background: #e7f5ff;
}
QHeaderView::section {
    background-color: #f8f9fa;
    color: #212529;
    padding: 8px;
    border: none;
    border-right: 1px solid #dee2e6;
    border-bottom: 1px solid #dee2e6;
    font-weight: bold;
}
QScrollBar:vertical {
    border: none;
    background: #f8f9fa;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #dee2e6;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::handle:vertical:hover {
    background: #adb5bd;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}
QScrollBar:horizontal {
    border: none;
    background: #f8f9fa;
    height: 10px;
    margin: 0px;
}
QScrollBar::handle:horizontal {
    background: #dee2e6;
    min-width: 20px;
    border-radius: 5px;
}
QScrollBar::handle:horizontal:hover {
    background: #adb5bd;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}
QPushButton#save_button {
    color: #fff;
    background-color: #007bff;
    border-color: #007bff;
    font-weight: bold;
}
QPushButton#save_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#save_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}</string>
       </property>
       <property name="text">
        <string>저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}</string>
       </property>
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
